pkg/controller/seed-controller-manager/initial-application-installation-controller/controller.go

```diff
@@ -224,3 +240,12 @@ func (r *Reconciler) removeAnnotation(ctx context.Context, cluster *kubermaticv1
 	delete(cluster.Annotations, kubermaticv1.InitialApplicationInstallationsRequestAnnotation)
 	return r.Patch(ctx, cluster, ctrlruntimeclient.MergeFrom(oldCluster))
 }
+
+func getCiliumApplicationInstallation(userClusterClient ctrlruntimeclient.Client) (*appskubermaticv1.ApplicationInstallation, error) {
+	app := &appskubermaticv1.ApplicationInstallation{}
+	if err := userClusterClient.Get(context.Background(), types.NamespacedName{Namespace: metav1.NamespaceSystem, Name: kubermaticv1.CNIPluginTypeCilium.String()}, app); err != nil {
```

Comment:
Can we please make sure that the context here is not `context.Background()` but `ctx` from the reconcile loop?